dts/bindings/cpu/arm,cortex-r5f.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2019 Stephanos Ioannidis <root@stephanos.io>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: ARM Cortex-R5F CPU
+
+description: >
+    This binding gives a base representation for ARM Cortex-R5F CPU.
```

Comment:
Could shorten to just be the same thing as the title.

Reply:
All the other bindings seem to use `This binding gives a base representation ...` or a similar long form in the description.

Is there a plan to change this format in a near future? If not, I believe, at least for the sake of uniformity, it should follow the form used by all the other bindings. 

Maybe it can be updated later when, if ever, such a change is implemented across the repo.

Reply:
I'm a "fight the flow" kind of guy personally ;). Not super important though. Might do a mass-fixup later.

Reply:
I don't even know what "base" is supposed to mean there to be honest. It's not like there are other bindings for it.

I suspect it meant something wherever it was added originally, and then it was copied from there to everywhere. Trying to get rid of mystery stuff like that long-term, so that it's obvious what everything is doing.

Reply:
> I don't even know what "base" is supposed to mean there to be honest. It's not like there are other bindings for it.

Fair point. It's not like this is a "base" binding that is included by other bindings (as it is done to `cpu.yaml`, for instance). I will update the description to a more generic one that goes like `This is a representation of ...`.

I still think the description field deserves more than just the title though; after all, that is the whole point of having a description field at all. At the same time, I do understand that it is rather pointless to have a description that simply states the very same thing as the title and does not enrich developer understanding of a specific binding as done in here.

The real problem here is that the description field must be present, and you are forced to fill it out regardless of whether you actually need it or not. Removing an unneeded description field will result in `devicetree error: missing 'description' property in *`.

Maybe, making the description field optional and allowing it to be specified only when it is required would be a more sensible approach to this problem, as it is done in Linux.